backend/test/richard_burton/author_test.exs

```diff
@@ -0,0 +1,77 @@
+defmodule RichardBurton.AuthorTest do
+  @moduledoc """
+  Tests for the Author schema
+  """
+
+  use RichardBurton.DataCase
+
+  alias RichardBurton.Author
+
+  @valid_attrs %{"name" => "J. M. Pereira da Silva"}
+
+  defp changeset(attrs) do
+    Author.changeset(%Author{}, attrs)
+  end
+
+  defp insert(attrs) do
+    attrs |> changeset() |> Repo.insert()
+  end
+
+  defp count do
+    Kernel.length(Author.all())
```

Comment:
NIT: The `Kernel` module is imported by default in every Elixir module so you can just do `length(Author.all())`